src/components/transport_manager/src/transport_manager_default.cc

```diff
@@ -41,7 +41,11 @@
 #endif
 
 #ifdef BLUETOOTH_LE_SUPPORT
-#include "transport_manager/bluetooth_le/bluetooth_le_transport_adapter.h"
+#include "transport_manager/android_ipc/android_transport_adapter.h"
+#endif
+
+#ifdef ANDROID_BT_SUPPORT
+#include "transport_manager/android_ipc/android_transport_adapter.h"
```

Comment:
@VadymLuchko I believe this can be updated:
```
#if defined(BLUETOOTH_LE_SUPPORT) || defined(ANDROID_BT_SUPPORT)
#include "transport_manager/android_ipc/android_transport_adapter.h"
#endif
```

Reply:
fixed in 3a0c69bbb2efefd761d10af521a1c260a62c0d6b